README.md

```diff
@@ -181,98 +232,699 @@ metasync.series(
-
-## Debounce function, delayed execution
-`metasync.debounce(timeout, fn, ...args)`
-- `timeout` - msec
-- `fn` - function to be debounced
-- `...args` - arguments for fn (optional)
-
-## Set timeout for asynchronous function execution
-`metasync.timeout(timeout, fn, callback)`
-- `timeout` - time interval
-- `fn` - async function to be executed
-- `callback` - callback function on done
-
-## Queue instantiation
-`metasync.queue(concurrency)`
-- `concurrency` - number of simultaneous and asynchronously executing tasks
-
-## Transforms function with args arguments and callback
-to function with args as separate values and callback
-`metasync.toAsync(fn)`
-- `fn` - function contract callback-last, callback contranct err-first
-- Returns: function with arguments gathered from args as separate values
-and callback
-
-## Async function
-`metasync.asAsync(fn, ...args)`
-- `fn` - function
-- `...args` - its arguments
-
-## Convert source to callback-last contract
-`metasync.callbackify(source)`
-- `source` - promise or regular synchronous function
-- Returns: callback, function
-
-## Convert async function to Promise object
-`metasync.promisify(func)`
-- `func:function` - callback-last function
-- Returns: object, Promise instance
-
-## Convert sync function to Promise object
-`metasync.promisifySync(func)`
-- `func:function` - regular synchronous function
-- Returns: object, Promise instance
+### every(items, fn, done)
+
+- `items`: [`<Array>`][array] incoming
+- `fn`: [`<Function>`][function]
+  - `value`: `<any>` item from items array
+  - `callback`: [`<Function>`][function]
+    - `err`: [`<Error>`][error]|[`<null>`][null]
+    - `accepted`: [`<boolean>`][boolean]
+- `done`: [`<Function>`][function] on done
+  - `err`: [`<Error>`][error]|[`<null>`][null]
+  - `result`: [`<boolean>`][boolean]
+
+Asynchronous every
+
+### some(items, fn, done)
+
+- `items`: [`<Array>`][array] incoming
+- `fn`: [`<Function>`][function]
+  - `value`: `<any>` item from items array
+  - `callback`: [`<Function>`][function]
+    - `err`: [`<Error>`][error]|[`<null>`][null]
+    - `accepted`: [`<boolean>`][boolean]
+- `done`: [`<Function>`][function] on done
+  - `err`: [`<Error>`][error]|[`<null>`][null]
+  - `result`: [`<boolean>`][boolean]
+
+Asynchronous some (iterate in series)
+
+### asyncMap(items, fn\[, options\]\[, done\])
+
+- `items`: [`<Array>`][array] incoming dataset
+- `fn`: [`<Function>`][function]
+  - `item`: `<any>`
+  - `index`: [`<number>`][number]
+- `options`: [`<Object>`][object] map params, optional
+  - `min`: [`<number>`][number] min number of items in one next call
+  - `percent`: [`<number>`][number] ratio of map time to all time
+- `done`: [`<Function>`][function] call on done, optional
+  - `err`: [`<Error>`][error]|[`<null>`][null]
+  - `result`: [`<Array>`][array]
+
+Non-blocking synchronous map
+
+### asyncIter(base)
+
+- `base`: [`<Iterable>`][iterable]|`<AsyncIterable>` an iterable that is wrapped
```

Comment:
We can link `AsyncIterable` to https://tc39.github.io/ecma262/#sec-asynciterable-interface since there is no MDN article for it.